content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/stock.rst

```diff
@@ -0,0 +1,112 @@
+============
+Stock report
+============
+
+Use the stock report in Odoo *Inventory* for a detailed list of all stored products, including those
+reserved, purchased and in transit, as well as those delivered to customers.
+
+.. note::
+   The reporting feature only accessible to users with :doc:`admin access
+   <../../../../general/users/access_rights>`.
+
+To access the stock report, go to :menuselection:`Inventory app --> Reporting --> Stock`.
+
+.. image:: stock/stock-report.png
+   :align: center
+   :alt: Show the stock report, accessible by going to Inventory > Reporting > Stock.
+
+Navigate the stock report
+=========================
+
+On the stock report, the left sidebar includes :guilabel:`Warehouses` to filter products by specific
+warehouses and :guilabel:`Category`, to show products within a selected product category.
```

Comment:
```suggestion
On the stock report, the left sidebar includes several groupings to narrow down what is being shown. The default groupigs are :guilabel:`Warehouses`,  which filters products by specific
warehouses, and :guilabel:`Category`, which shows products within a selected product category.
```

I would also add a note that the Wareouses only shows up if in a ulti=warehouse nevironment. In Runbot I only see the categories.